Makefile

```diff
@@ -57,10 +57,10 @@ gen-server-protobuf:
 
 gen-load-test-protobuf:
 	protoc \
-		--elixir_out=./load_test/lib/load_test/ \
+		--elixir_out=./load_test/lib/load_test/communication \
```

Comment:
Is this change necessary? Or is it working properly now?

Reply:
It is